apps/client/src/app/capability-statement-wrapper/r4/capability-statement.component.ts

```diff
@@ -57,6 +55,33 @@ export class R4CapabilityStatementComponent extends BaseComponent implements OnI
     this.capabilityStatement = new CapabilityStatement({ meta: this.authService.getDefaultMeta() });
   }
 
+  getCodes(list: any, resourceIndex) {
+    if(list != null) {
+      const remainingCodes = [];
+      this.codes.forEach((coding, index) => {
+        const listIndex = list.findIndex((element) => {
+          if (element.type === coding.code) {
+            return true;
+          }
+        })
+        if (listIndex === resourceIndex || listIndex === -1) remainingCodes.push(coding);
+      });
+      return remainingCodes;
+    }
+  }
+
+
+  public defaultResource(list: any) {
+    if(list != null) {
+      const remainingCodes = [];
+      this.codes.forEach((coding, index) => {
```

Comment:
This is fine and works as-is, but consider in the future using `this.codes.filter((coding, index) => { })`